hack/ci/e2e-k8s.sh

```diff
@@ -265,11 +265,11 @@ main() {
   kind version
 
   # build kubernetes
-  if [ "${BUILD_TYPE:-}" = "bazel" ]; then
-    build_with_bazel
-  else
+  #if [ "${BUILD_TYPE:-}" = "bazel" ]; then
```

Comment:
older branches? and/or delete completely?

Reply:
leaving the most trivial revertable state, will come back and fully delete these code paths from more than just the script once I get a chance to confirm the state of CI.